### DISCUSSION TITLE
Any active forks elsewhere?

### DISCUSSION THREAD

**Discussion by 0xeb** [OpenSalamander/salamander#54] [General]
I am curious if there are active forks elsewhere. Clearly the owners of this repo are super busy and it makes no sense to make PRs here.

**Comment by melloware** [comment]
You can see the forks here: https://github.com/OpenSalamander/salamander/forks  But none of them seem too active yet.

**Comment by janrysavy** [comment]
I'm sorry, I'm still finishing my work. After that, I plan to invest some time in Salamander development. The question is how to proceed with the translation, as there is no one willing to do a review yet and the automatic translation would need it.

**Reply by melloware** [reply]
@janrysavy would it really be the that bad if the translation was just accepted as is even if some are slightly incorrect. I feel like this project will never move forward and the perfectness of comments should not hold back progress.  Looking forward to this project to get going.

**Comment by eric-jesover** [comment]
for me automatic translation could merge to start to have real work starting

**Reply by melloware** [reply]
I agree.  I think the automated translation is a great start even if its not perfect.

**Comment by KRtekTM** [comment]
In one email I wrote to @janrysavy that I cannot guarantee 100% code quality in the features I started working on (as was shown by the slowdown in application startup after implementing tabbed panels), so I am inclined not to mix the original source code and changes prepared completely using artificial intelligence. That is why I decided to make a fork for brave users - it has bugs and it may take some time to fix them.

It will be called Open Salamander: Samandarin (samandarin is the natural toxin secreted by fire salamander, symbolizing a spicy, daring twist — just like my AI-crafted changes). The Samandarin experimental fork continues the story by blending the original Salamander DNA with AI-driven enhancements, keeping development moving while inviting adventurous contributors to explore and extend the project.

It's published: https://github.com/KRtkovo-eu-AI/salamander/releases/tag/5.0-samandarin-0.1

<img width="1424" height="734" alt="image" src="https://github.com/user-attachments/assets/2819443b-c043-435a-81a6-76159821dd2d" />

**Reply by KRtekTM** [reply]
FIY: I haven't stopped developing experimental features for the Samandarin fork, I just have less free time now - I'll have fun with it again during long winter evenings :)
I would like to release version 5.0-samandarin-0.2 sometime around Christmas or New Year, it may only contain some minor changes, but the project is definitely not sleeping, I am still thinking about it.

**Reply by Akrotkiv** [reply]
I am very curious for some release. I was delighted by your post. 
Unfortunately not shure  cause total virus webpage reported few suspects on both .zip and .exe releases.
![2025-11-03_225623](https://github.com/user-attachments/assets/839aaa23-e22a-4cf9-9cc0-eb636397bfd3)
![2025-11-03_225758](https://github.com/user-attachments/assets/d470ada3-92ba-43cd-b511-f5a54a06bb25)

**Reply by KRtekTM** [reply]
I know about it, it looks like a false flag, please see #273 which is related to this issue.

**Reply by Akrotkiv** [reply]
OK. As I read this https://stackoverflow.com/questions/75886428/fake-positive-bit-defender-problem-genvariant-tedy-304469,
I believe other big antivirus brands that approved it as clean.
My desire for a functional salamander, or samandarin, is greater than my safety paranoia :)
I am going to try.

**Reply by KRtekTM** [reply]
It's good to be a bit paranoid these days, especially after the SunBurst incident (where the build agents were compromised as a part of a supply chain attack). However, I really think this is a combination of a missing digital signature and a fake evaluation.
To increase security assurance for the future `5.0-samandarin-0.2` release build, I'll use an auditable virtual machine that will be quarantined without Internet access.
